torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -271,6 +274,33 @@ Operation createUnaryOp(
   };
 }
 
+void MKLDNNLayerNormOp(Stack* stack) {
+  c10::impl::ExcludeDispatchKeyGuard edkg(c10::autograd_dispatch_keyset);
+
+  // enable_cudnn not used
+  pop(stack);
+  auto eps = pop(stack).toDouble();
+
+  Tensor bias{};
```

Comment:
nit: `Tensor bias{}` is pretty weird syntax, and takes me a bit to parse reading through the code. Can you just move the initialization down to `Tensor bias = bias_ival.toTensor()` ? (same for weight)